Examples/Tests/scraper/PICMI_inputs.py

```diff
@@ -0,0 +1,209 @@
+# --- Input file for MCC testing. There is already a test of the MCC
+# --- functionality so this just tests the PICMI interface
+
+import numpy as np
+from pywarpx import picmi
+import pywarpx
+
+constants = picmi.constants
+
+##########################
+# physics parameters
+##########################
+
+D_CA = 0.067 # m
+
+N_INERT = 9.64e20 # m^-3
+T_INERT = 300.0 # K
+
+FREQ = 13.56e6 # MHz
+
+VOLTAGE = 450.0
+
+M_ION = 6.67e-27 # kg
+
+PLASMA_DENSITY = 2.56e14 # m^-3
+T_ELEC = 30000.0 # K
+
+DT = 1.0 / (400 * FREQ)
+
+##########################
+# numerics parameters
+##########################
+
+# --- Number of time steps
+max_steps = 10
+diagnostic_intervals = "::5"
+
+# --- Grid
+nx = 128
+ny = 16
+
+xmin = 0.0
+ymin = 0.0
+xmax = D_CA
+ymax = D_CA / nx * ny
+
+number_per_cell_each_dim = [32, 16]
+
+##########################
+# physics components
+##########################
+
+v_rms_elec = np.sqrt(constants.kb * T_ELEC / constants.m_e)
+v_rms_ion = np.sqrt(constants.kb * T_INERT / M_ION)
+
+uniform_plasma_elec = picmi.UniformDistribution(
+    density = PLASMA_DENSITY,
+    upper_bound = [None] * 3,
+    rms_velocity = [v_rms_elec] * 3,
+    directed_velocity = [0.] * 3
+)
+
+uniform_plasma_ion = picmi.UniformDistribution(
+    density = PLASMA_DENSITY,
+    upper_bound = [None] * 3,
+    rms_velocity = [v_rms_ion] * 3,
+    directed_velocity = [0.] * 3
+)
+
+electrons = picmi.Species(
+    particle_type='electron', name='electrons',
+    initial_distribution=uniform_plasma_elec
+)
+ions = picmi.Species(
+    particle_type='He', name='he_ions',
+    charge='q_e',
+    initial_distribution=uniform_plasma_ion
+)
+electrons.scrape_zmin = 1
+electrons.scrape_zmax = 1
+electrons.scrape_xmax = 1
+electrons.scrape_xmin = 1
+
+ions.scrape_zmin = 1
+ions.scrape_zmax = 1
+ions.scrape_xmax = 1
+ions.scrape_xmin = 1
+
+# MCC collisions
```

Comment:
Adding MCC to this script is not needed since we don't care to test it here, so please remove the collisions.